test_aodncore/pipeline/test_dummyHandler.py

```diff
@@ -109,43 +109,124 @@ def test_notify_error(self, mock_smtp):
         mock_smtp.return_value.sendmail.return_value = {}
 
         handler = self.run_handler_with_exception(ComplianceCheckFailedError, NOT_NETCDF_NC_FILE,
-                                                  notify_params={'error_notify_list': ['email:nobody1@example.com',
-                                                                                       'email:nobody2@example.com']},
+                                                  notify_params={'notify_owner_error': False,
```

Comment:
Ok now I'm _really_ just nitpicking, but the naming of these boolean params just looks a bit wrong to me. Either 'owner_notify_error' or 'error_notify_owner' would look less confusing to me (and similar for the success ones)

Reply:
Nitpicking is relevant here, because it should make sense to a "user".

I think I was going for "notify [the] owner [of] error" and "notify [the] owner [of] success", but happy with whatever makes the most sense.

Reply:
Sorry... I agree that makes sense in itself, I'm just comparing it to the other notify_params, but it reallly doesn't matter that much. Never mind.